modules/tv/src/main/TvSyncActor.scala

```diff
@@ -51,12 +53,17 @@ final private[tv] class TvSyncActor(
 
     case lila.core.game.StartGame(g) =>
       if g.hasClock then
-        val candidate = Tv.Candidate(g, g.userIds.exists(lightUserApi.isBotSync))
-        channelActors
-          .collect {
-            case (chan, actor) if chan.filter(candidate) => actor
-          }
-          .foreach(_.addCandidate(g))
+        userApi
+          .countEngines(g.userIds)
```

Comment:
this adds a hit to the database every time a game starts, so about 5 million requests a day.

Reply:
Could you propose an alternative solution? I am mostly asking out of curiosity as I am trying to get a better understanding of the codebase.